### ISSUE TITLE
Argument 1 passed to craft\web\View::renderTemplate() must be of the type string, null given

### ISSUE THREAD

**Issue by romainpoirier** [barrelstrength/craft-sprout-forms#600]
### Description (screenshots, logs, ...)
Using the native render form method:
`{{ craft.sproutForms.displayForm('contact') }}`

I get this error:
`Argument 1 passed to craft\web\View::renderTemplate() must be of the type string, null given, called in /vendor/barrelstrength/sprout-base-email/src/base/EmailTemplates.php on line 145`

While there is a correct template path in `/admin/sprout-forms/settings` > `Default Form Templates` field.

### Additional info
- Craft version: Craft Pro 3.7.34
- PHP version: 7.4.21
- Database driver & version: MySQL 5.7.34
- Plugins & versions: 7.4.21

**Comment by BenParizek** [comment]
@romainpoirier The error your posting looks like it's coming from a misconfiguration of the Email Templates, not the Form Templates. For next steps I'd consider:

1. Confirm your form data is being saved to the database
2. Check your notification email settings for this form and confirm the templates being used there are accurate
3. If things still aren't working after that, try disabling the Notification Emails for this form and try to confirm which Notification is the issue

Let me know what you find and we can dig in more if needed.

**Comment by romainpoirier** [comment]
> Confirm your form data is being saved to the database

Yes, despite the error it was being saved and I can retrieve the submissions under `/admin/sprout-forms/entries`.

> Check your notification email settings for this form and confirm the templates being used there are accurate

Ok I finally don't get this error anymore. It seems that it was maybe related to the template email path.

In my Craft 2 project my email template was located here:

> /templates/emails/notification.twig

While in Craft 3 it should be located here:

> /templates/emails/notification/email.twig

**Comment by romainpoirier** [comment]
Unfortunately, I now get this error while I try to preview the email template from `/admin/sprout-email/notifications/edit/{id`}:

> Twig Template Loading Error – craft\web\twig\TemplateLoaderException
> Impossible to find the template « _emails/notification » in "_emails/notificationAdmin/email.twig" at line 1.

While the file belongs at the correct path. Submissions are saved, but notifications never sent.

**Comment by romainpoirier** [comment]
`_emails/notification` is the default email template, which could be setup from `/admin/sprout-forms/settings` → `Default Form Templates` but is useless in my case.

In parallel, for my contact form, I have a `_emails/notificationAdmin` and a `_emails/notificationVisitor` templates folder, because I'm sending two emails on submit. In both folders I have an `email.twig` and `email.txt` template. These templates are setup at /admin/sprout-email notifications → `Templates` dropdown.
